keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -883,8 +883,9 @@ clusterset, associated with the derived `ServiceImport`. One or more
 `EndpointSlice` containing only endpoints from a single source cluster. These
 `EndpointSlice` objects will be marked as managed by the clusterset service
 controller, so that the endpoint slice controller doesn’t delete them.
-`EndpointSlices` will have an owner reference to their associated
-`ServiceImport`.
+`EndpointSlices` should have an owner reference to their associated
+`ServiceImport` unless the implementation has another mechanism to delete
```

Comment:
this seems very open, are there any requirements this "mechanism to delete EndpointSlices" should met?

Reply:
I guess the only requirement is that the EndpointSlice should be deleted when the service is unexported. Exactly how that is done shouldn't matter. In fact, two known implementations do not use owner refs, hence this PR.

How about this text:

_When a service is un-exported, the associated EndpointSlices will be deleted. The specific mechanism by which they are deleted is an implementation detail._